.github/workflows/actions.yaml

```diff
@@ -0,0 +1,23 @@
+name: Run Flink Jira Bot Hourly
+
+on:
+  schedule:
+      - cron: 0 * * * *
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9]
```

Comment:
Is there a reason to run the script with a strategy matrix? Maybe it would make more sense to encode the python version in the `setup-python` step, since I don't imagine you'll want to run this in parallel on different python versions?

Reply:
You are right. I was just lazy.